src/applications/vaos/covid-19-vaccine/components/ContactFacilitiesPage.jsx

```diff
@@ -73,14 +73,14 @@ export default function ContactFacilitiesPage() {
     <div>
       {canUseVaccineFlow && (
         <>
+          <h1>{pageTitle}</h1>
           <AlertBox
             className="vads-u-margin-top--0"
-            level="1"
+            level="2"
             status="warning"
             backgroundOnly
-            headline={pageTitle}
+            headline="If you got your first dose:"
```

Comment:
In the design, it doesn't look like this should be a header/headline, but might want to check with Lauren or Peter, if you haven't.

Reply:
Just checked with Peter. Said he prefers having a header in the alert box.